### PR TITLE
Update esthemes.sh

### PR DESCRIPTION
**Pull request by CrashCortez**
Add 1 new theme
Crashstheme

### CODE REVIEW

File: scriptmodules/supplementary/esthemes.sh [hunk 1]
```diff
@@ -185,6 +185,7 @@ function gui_esthemes() {
         'waweedman Venom'
         'waweedman Spider-Man'
         'blowfinger77 locomotion'
+        'CrashCortez Crashstheme'
```
Comment (cmitu) [inline scriptmodules/supplementary/esthemes.sh]: For this to work, you'll have to name your theme Github repository `es-theme-crashstheme`.

**Comment by hhromic** [comment]
> This theme is based off of Both Tronkyfran and Carbon, I loved the main menu pictures of TronkyFran, and the Game menus of Carbon.

@CrashCortez , did you get proper permission from the respective authors? It is not mentioned on your README file.

From the Tronkyfran theme conditions:
```
ALLOWED:
Share and duplicate as it is
Edit, alter, change it

REQUIREMENTS:
Attribution, give credit to the creator
Indicate changes to it
Publish the changes under the same license

PROHIBITED:
Commercial distribution
```

From the Carbon theme conditions:
```
Summary of the license below:

ALLOWED:      - Share and duplicate as it is
              - Edit, alter, change it
REQUIREMENTS: - Attribution, give credit to the creator
              - Indicate changes to it
              - Publish the changes under the same license
PROHIBITED:   - Commercial distribution
```

**Comment by hhromic** [comment]
@CrashCortez also, are you one of the moderators of the RetroPie "Official" Facebook page?

File: scriptmodules/supplementary/esthemes.sh (hunk 1 above)
Comment (CrashCortez) [reply scriptmodules/supplementary/esthemes.sh]: ill fix that now, thank you.

**Comment by cmitu** [comment]
Good for you for helping people, bad for misleading them with the _Official_ Retropie FB page.

**Comment by CrashCortez** [comment]
I didn't name it, but i can ask that the name be changed.

**Comment by CrashCortez** [comment]
Actually I removed myself as a moderator

**Comment by CrashCortez** [comment]
Also added the licence and the proper credits to the theme readme. let me know if I need to adjust anything else. Otherwise no hard feelings if I don't get approved love the work, and I want you guys have a great weekend and happy gaming.

**Comment by joolswills** [comment]
You're not welcome here if you are in anyway linked to that FB group.

**Comment by CrashCortez** [comment]
Fair enough, I’ll pull all my work on the retropie stuff I’ve been working on. 

I wish you all good luck on your future works.

**Comment by techiedj** [comment]
SO ... I've been a loooong time lurker and supporter of Rpi and retro, 4 Pi's 3 of Wich are retro. I like piHole and rtlsdr as well but this right here frankly pisses me off. If it wasn't for said group there would be no retro tinkerboard, simple period.  I understand your the maintainer of the almighty pie and don't understand your beef with ROTT, but I appreciate anyone and everyone that contributes to the cause so put up or shut up! Tinker or die! /No Carrier

**Comment by joolswills** [comment]
I have no problem with people supporting the tinker board. I do have an issue with a FB group that advertises itself as the official group and associates RetroPie with piracy.

**Comment by CrashCortez** [comment]
It’s all good, like I said I removed my self from the group and will be moving on to other projects. While I was just helping people in said group, I do see/and understand Retropie (and their devs) point on this. Before I left I suggested that they change the name to stop any confusion. 

I wish all the best to RetroPie and the devs, and while a little salty over the experience, I have no hard feelings. 

Anyone reading this and want to post about anything regarding this, I wouldn’t, just move on. 

Thanks agian “RetroPie” and all your hard work.

**Comment by joolswills** [comment]
Thanks. I appreciate that you are not longer involved in the group. I have no problem with you personally or with your contributions if you are not linked to that page.

Not directed to you, but for anyone reading this thread: I hope people can understand the large amount of time I have to spend due to misuse of our project name and licence(s). These premade images are often included with people selling RetroPie also which damages the projects name and reputation. Users who install them will not get any support on our site, and they open themselves up to risks of malware.

**Comment by techiedj** [comment]
Cooler heads prevail!  Appreciate everyone taking the high road and apologize for my misunderstanding thinking this was directed at ROTT.  I can agree having issue with the "official" claim and glad I wasn't even aware of such shady group.

Live long and Retro! <3

**Pull request closed without merging**